tests/baselines/reference/user/puppeteer.log

```diff
@@ -1,72 +1,66 @@
 Exit Code: 1
 Standard output:
-lib/Browser.js(71,49): error TS2322: Type 'Promise<CDPSession>' is not assignable to type 'Promise<Puppeteer.CDPSession>'.
```

Comment:
Seems like the results of the submodule update too, but I need to look at the diff using beyond compare (github missed all the nearly-identical lines)